package.json

```diff
@@ -9,7 +9,7 @@
     "watch-test": "karma start --auto-watch --no-single-run",
     "pack": "cross-env NODE_ENV=production webpack -p --progress",
     "pack-nightly": "NODE_ENV=nightly webpack --progress",
-    "start": "cross-env NODE_ENV=development webpack-dev-server --progress --host 0.0.0.0 --hot --https"
+    "start": "webpack-dev-server --mode development --watch"
```

Comment:
@chizhovm Please comment how the changes relate to this task ?

Reply:
This change relate https://github.com/diglabby/mapa/issues/340. I think it's my mistake to introduce it into this PR. Without this change I couldn't start the project.

Reply:
@chizhovm please move the changes out of the commit. 
They are very important but they relates to another issue. 

So please think we push into Pull Request the only code we need to solve the issue. For all other code we can open a new issue and do the same. Such a division helps to monitor code changes perfectly.